modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -816,6 +816,23 @@ public void postProcessParameter(CodegenParameter parameter) { }
               itr.remove();
             }
         }
+
+        @SuppressWarnings("unchecked")
+        Map<String, Object> operations = (Map<String, Object>) objs.get("operations");
+        if (operations != null) {
+
+            @SuppressWarnings("unchecked")
+            List<CodegenOperation> ops = (List<CodegenOperation>) operations.get("operation");
+            for (CodegenOperation operation : ops) {
+
+                if (operation.returnType == null) {
+                    operation.returnType = "void";
```

Comment:
No problem lets move it to jaxrs first and then do it one by one for each language.
So I can scope it for jaxrs-cxf here and move it to the CXF language?